alibi/explainers/cem.py

```diff
@@ -40,46 +40,46 @@ def __init__(self,
         Parameters
         ----------
         predict
-            Keras or TensorFlow model or any other model's prediction function returning class probabilities
+            `Keras` or `TensorFlow` model or any other model's prediction function returning class probabilities.
```

Comment:
I would remove `Keras` as this was before `keras` and `tensorflow` were different libraries, applies to CEM, CI and CFP.